libusb/hid.c

```diff
@@ -182,7 +182,7 @@ static int return_data(hid_device *dev, unsigned char *data, size_t length);
 
 static hid_device *new_hid_device(void)
 {
-	hid_device *dev = calloc(1, sizeof(hid_device));
+	hid_device *dev = (hid_device*) calloc(1, sizeof(hid_device));
```

Comment:
That would be good if you stick to one notation - either use `(hid_device*)` or `(hid_device *)` at all places. Otherwise, this looks good